installers/vmware/kickstart-esxi.go

```diff
@@ -308,18 +301,13 @@ kickstartpishellcmd=$(custom_data "['kickstart']['postinstall_shell_cmd']")
 
 # Kickstart postinstall supplemental config URL
 if [ "$kickstartpiurl" != "null" ]; then
```

Comment:
Should we also ignore empty urls?

Reply:
We do want to check to make sure we dont try to wget when there isn't a URL provided.

Reply:
We shouldn't be ignoring empty URLs no.